doc/postman/device-onvif-camera.postman_collection.json

```diff
@@ -1,10 +1,10 @@
 {
 	"info": {
-		"_postman_id": "117aa9ba-01bf-421f-bf59-fb109888c2d4",
-		"name": "device-onvif-camera Copy 4",
+		"_postman_id": "41b06358-d096-4b9b-8735-a0905762330f",
+		"name": "device-onvif-camera Copy 2",
```

Comment:
should this be renamed to `device-onvif-camera`?

Reply:
Yes, good catch